modules/r2b2AnalyticsAdapter.js

```diff
@@ -0,0 +1,574 @@
+import {ajax} from '../src/ajax.js';
+import adapter from '../libraries/analyticsAdapter/AnalyticsAdapter.js';
+import {EVENTS} from '../src/constants.js';
+import adapterManager from '../src/adapterManager.js';
+import {getGlobal} from '../src/prebidGlobal.js';
+import {logWarn, logError, isNumber, isStr, isPlainObject} from '../src/utils.js';
+import {getRefererInfo} from '../src/refererDetection.js';
+import {config} from '../src/config.js';
+
+const ADAPTER_VERSION = '1.1.0';
+const ADAPTER_CODE = 'r2b2';
+const MODULE_NAME = 'R2B2 Analytics'
+const GVLID = 1235;
+const analyticsType = 'endpoint';
+
+const DEFAULT_SERVER = 'log.r2b2.cz';
+const DEFAULT_EVENT_PATH = 'prebid/events';
+const DEFAULT_ERROR_PATH = 'error';
+const DEFAULT_PROTOCOL = 'https';
+
+const ERROR_MAX = 10;
+const BATCH_SIZE = 50;
+const BATCH_DELAY = 500;
+const REPORTED_URL = getRefererInfo().page || getRefererInfo().topmostLocation;
```

Comment:
Could we add a default empty string here, in the case these both happen to come back as undefined?